CHANGELOG.md

```diff
@@ -2,59 +2,65 @@
 
 ## Unreleased
 
+This version updates nhsuk-frontend to version 9.
```

Comment:
probably worth writing that this version provides support for version 9, rather than updates